recipes/oommf/bld.bat

```diff
@@ -0,0 +1,5 @@
+curl -o oommf.zip  "http://math.nist.gov/oommf/dist/oommf12b0_20160930_86_x64.zip"
```

Comment:
Looks like the [move command](https://ss64.com/nt/move.html) is equivalent to `mv` in Unix.